js/handlers.js

```diff
@@ -29,6 +29,15 @@ weechat.factory('handlers', ['$rootScope', '$log', 'models', 'plugins', 'notific
         new_date.setHours(0, 0, 0, 0);
         // Check if the date changed
         if (old_date.valueOf() !== new_date.valueOf()) {
+            var supports_formatting_date = false;
+            var should_be_tuesday = new Date(Date.UTC(2012,11,12,3,12,0))
+                .toLocaleDateString("en-US-u-ca-iso8601", {"weekday":"long"});
```

Comment:
The alternative given on [the MDN page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString#Checking_for_support_for_locales_and_options_arguments) was to check if the right exception is thrown when giving an invalid index. We could do this, but to me that doesn't guarantee that the date will be formatted correctly. I'm just assuming less about a browser in the wild by checking what I'm actually trying to check.

As for returning Wednesday, I think that might be UTC mixup. If we want to keep it as this code, `new Date(new_date.getTimezoneOffset()*60*1000).toLocaleDateString("en-US-u-ca-iso8601", {"weekday":"long"})'` should return Tuesday. If we want to not keep this code and go with the function that the MDN page has, we can do that as well. (But I can't see anything we're going to do be not hacky).

As for running it once on gb loading, where do you suggest putting that?